app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -127,8 +130,8 @@ void addItem() {
         }
     }
 
-    @OnItemClick(R.id.content_list)
-    void showNameOptions(final int position) {
+    @Override
+    public void showNameOptions(final int position) {
         nameEditChoicesDialog.showChoices(namesAdapter.getItem(position));
```

Comment:
Hmmm, now that I think about this, having the adapter fire this callback and then have the fragment read from the adapter seems kind of roundabout. Let's just pass the name here directly.